src/lines.jl

```diff
@@ -9,12 +9,12 @@ function graphline(g, locs_x, locs_y, nodesize::Vector{T}, arrowlength, angleoff
         j = dst(e)
         Δx = locs_x[j] - locs_x[i]
         Δy = locs_y[j] - locs_y[i]
-        d  = sqrt(Δx^2 + Δy^2)
-        θ  = atan(Δy,Δx)
+        d = sqrt(Δx^2 + Δy^2)
+        θ = atan(Δy,Δx)
         startx = locs_x[i] + nodesize[i]*cos(θ)
         starty = locs_y[i] + nodesize[i]*sin(θ)
-        endx  = locs_x[i] + (d-nodesize[j])*1.00*cos(θ)
-        endy  = locs_y[i] + (d-nodesize[j])*1.00*sin(θ)
+        endx = locs_x[j] + nodesize[j]*cos(θ+π)
+        endy = locs_y[j] + nodesize[j]*sin(θ+π)
```

Comment:
What is the reason for adding Pi here? Isn't it the case, that `cos(x + π) == -cos(x)` and `sin(x + π) == -sin(x)` ?

Reply:
> Isn't it the case, that cos(x + π) == -cos(x) and sin(x + π) == -sin(x) ?

Yes, this is true.

```julia
julia> using Test
julia> for θ in 0:45:360
       @test sind(θ+180) == -sind(θ)
       @test cosd(θ+180) == -cosd(θ)
       end
```

> What is the reason for adding Pi here?
The reason I added pi here is so that the structure of the code is concordant with how I was thinking about the problem. I framed the problem as being at the jth point, looking back at the ith point (theta+pi), then moving from the jth point toward the ith point (+sin or +cos).